.github/workflows/container-tests.yml

```diff
@@ -54,16 +25,61 @@ jobs:
         with:
           ref: "refs/pull/${{ github.event.issue.number }}/head"
 
+      - name: Set variables for Testing Farm
+        id: vars
+        run: |
+          api_key=${{ secrets.TF_INTERNAL_API_KEY }}
+          branch="master"
+          tf_scope="private"
+          tmt_repo="https://gitlab.cee.redhat.com/platform-eng-core-services/sclorg-tmt-plans"
```

Comment:
I would probably move these lines in the "else" section (now line 49), but it's OK by me even like this.